README.md

```diff
@@ -511,6 +514,55 @@ name:      lcd_1602_i2c
 web site:  https://github.com/raspberrypi/pico-examples/tree/HEAD/i2c/lcd_1602_i2c
 ```
 
+## verify
+
+This command allows you to verify the contents of Flash/SRAM on the device matches a file, similar to the `--verify` option for `save` and `load`.
+
+```text
+$ picotool help verify
+VERIFY:
+    Check that the device contents match those in the file.
+
+SYNOPSIS:
+    picotool verify [device-selection]
+
+OPTIONS:
+    Target device selection
+        --bus <bus>
+            Filter devices by USB bus number
+        --address <addr>
+            Filter devices by USB device address
+        --vid <vid>
+            Filter by vendor id
+        --pid <pid>
+            Filter by product id
+        --ser <ser>
+            Filter by serial number
+        -f, --force
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be rebooted back to application mode
+        -F, --force-no-reboot
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be left connected and accessible to picotool, but without the
+            RPI-RP2 drive mounted
+    The file to compare against
+        <filename>
+            The file name
+        -t <type>
+            Specify file type (uf2 | elf | bin) explicitly, ignoring file extension
+    Address options
+        -r, --range
+            Compare a sub range of memory only
+        <from>
+            The lower address bound in hex
+        <to>
+            The upper address bound in hex
+        -o, --offset
+            Specify the load address when comparing with a BIN file
+        <offset>
+            Load offset (memory address; default 0x10000000)
```

Comment:
Just a suggestion, but I wonder if this would be slightly clearer as:
```
        -o, --offset <offset>
            Specify the load address when comparing with a BIN file
            <offset>
                Load offset (memory address; default 0x10000000)
```
with `--range <from> <to>` being updated similarly?

Reply:
Yeah, the CLI stuff is all a bit weird - this is not a change I know how to make.

Reply:
In hindsight (having looked at the other help sections), perhaps the "problem" here is that `picotool verify [device-selection]` is simply too compact, and isn't showing all the available options being used, in contrast to e.g. `picotool erase -r <from> <to> [device-selection]` ? :thinking: